git-fs.c

```diff
@@ -27,6 +29,20 @@ bool enable_debug = 0;
 
 #define error(...) fprintf(stderr, __VA_ARGS__)
 
+// Dump a stacktrace to stderr
+static void dump_trace(int signum) {
+	error("Segmentation fault\n");
+	void * buffer[255];
+	const int calls = backtrace(buffer, sizeof(buffer) / sizeof(void *));
```

Comment:
:-) Thanks for taking a look, though.

I should probably just use the lengthof macro defined above here, for extra clarity...